### PR TITLE
Week 8 - Portfolio by Camilla Hallberg

### PR DESCRIPTION
**Pull request by CamillaHallberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+
+This project was not "supposed to be" that heavy on the tech side but I must admit that is has been a very great challenge to lay out the structure and then styling it. Since I will not take part in the career coaching programme I decided to not copy every design aspect from the Figma file. I am still very proud of the way my portfolio currently looks. I think it's a great start and I look forward to refining it along the course and as my programming skills deepen. 
+
+If I had even more time to focus on this project I would put more effort into accessibility, something I didn't manage to really dig into (which I find sad because I do find this aspect to be of such a great importance). I have run my code through both the W3C Markup Validation Service and the one for CSS and they have both passed (I had a few expected errors in markup that I corrected). I have also looked at my code in the most common browsers (Chrome, Safari, Edge and Firefox) and I think they look and function the same. 
+
+## View it live
+
+Here's the link for my portfolio: 
+https://portfolio-by-camilla.netlify.app/
+
```
Comment (faypi) [inline README.md:15]: Awesome README file. Made me pumped to see your code :)

File: code/style.css [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        margin: 4px;
+    }
+
+    .header-image-credit {
+        font-size: 14px;
+    }
+
+    .header-text h1 {
+        font-size: 80px;
+        width: 50px;
```
Comment (faypi) [inline code/style.css]: You can remove the width :)

File: code/style.css [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+    border-radius: 50%;
+    position: absolute;
+    top: 356px;
+    right: 24px;
+}
+
+.header-text {
+    position: relative;
+    top: 161px;
+    line-height: 60px;
```
Comment (faypi) [inline code/style.css]: You can increase this slightly. I checked it on desktop and you might need some extra breathing room :)

File: code/style.css [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    letter-spacing: 2px;
+}
+
+.my-thoughts-credit {
+    font-size: 12px;
+    text-align: end;
+    font-style: italic;
+}
+
+.skills-groups {
```
Comment (faypi) [inline code/style.css]: You can add some padding here like `padding: 30px;` so that you can see the bullet points on mobile!  :)

**Review by faypi — COMMENTED** [review]
Very nice looking website Camilla :D I appreciate the detailed README file! 
Great work with the design implementation and the color combo! 
Can't wait to see how this will evolve <3

File: README.md (hunk 1 above)
Comment (CamillaHallberg) [reply README.md:15]: Thank you, Fay! This makes me so happy! :)

File: code/style.css (hunk 3 above)
Comment (CamillaHallberg) [reply code/style.css]: Ahh, nice! Thanks for the suggestion! I'm increasing it to 65 px (would maybe like to do a bit more but then it starts to "interfere" with my profile image. 😅

File: code/style.css (hunk 4 above)
Comment (CamillaHallberg) [reply code/style.css]: Thanks! I actually noted this while checking mobile view after the hand-in and was wondering if someone would catch it... glad you did ;) Will change it :)

**Pull request closed without merging**